Mathlib/FieldTheory/Cardinality.lean

```diff
@@ -33,8 +33,9 @@ open scoped Cardinal nonZeroDivisors
 
 universe u
 
-/-- A finite field has prime power cardinality. -/
-theorem Fintype.isPrimePow_card_of_field {α} [Fintype α] [Field α] : IsPrimePow ‖α‖ := by
+/-- A finite division ring has prime power cardinality. -/
+theorem Fintype.isPrimePow_card_of_divisionring {α} [Fintype α] [DivisionRing α] :
```

Comment:
Many of these look potentially useful, but note that this one isn't strictly speaking a generalization since all finite division rings are fields.

Reply:
Reverted, thanks.